packages/pds/src/api/app/bsky/actor/createScene.ts

```diff
@@ -9,7 +9,7 @@ import AppContext from '../../../../context'
 
 export default function (server: Server, ctx: AppContext) {
   server.app.bsky.actor.createScene({
-    auth: ctx.accessVerifier,
+    auth: ctx.accessVerifierCheckTakedown,
```

Comment:
should we be checking this on every procedure? for instance CRUD routes

Reply:
wait actually, why is this necessary? looks like refresh tokens get revoked & they can't create a new session 🤔

Reply:
In my opinion it's not necessary on read routes, but I'm open to it.  It's a plus that the access tokens are stateless are don't require a db roundtrip to verify.  The irrevocability of the access tokens are intended to be balanced-out by them also being short-lived.